templates/embedding/index.html

```diff
@@ -0,0 +1,309 @@
+{% extends "embedding/base_embedding.html" %}
+
+{% block title %}Ubuntu is the new standard for embedded Linux{% endblock %}
+
+{% block meta_description %}Ubuntu Embedding Programme offers the optimal technical, commercial and legal framework to deploy your products and solutions with Canonical offerings.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1evAFIXbNztOd5yZee-uV8_OM-s6o27P46yXFaaeEhy0/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-6">
+      <h1>Canonical Embedding programme</h1>
+      <p>Our Embedding programme offers the optimal technical, commercial and legal framework to deploy your products and solutions with Canonical offerings, including cost-effective support across the lifecycle of your product.</p>
+      <p>
+        <a class="p-button--positive js-invoke-modal" href="/embedding#get-in-touch">Get in touch</a>
+      </p>
+    </div>
+    <div class="col-6 u-hide--small u-hide--medium u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/71baccc3-ubuntu-advantage-circular.svg",
+        alt="",
+        width="278",
+        height="300",
+        hi_def=True,
+        loading="lazy"
```

Comment:
```suggestion
        loading="auto"
```
As this will be visible on page load